enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```diff
@@ -208,11 +210,20 @@ enabled cluster. New guidelines would include the following to use CCO has detec
 
 - Changing CSV to add CredentialRequest RBAC
 - If loading the CredentialRequests from a yaml, place it into a discoverable location in the codebase
-- Add a bundle annotation claiming token-based authentication support
+- Add a bundle annotation claiming token-based authentication support:
+    ```yaml
+    apiVersion: operators.coreos.com/v1alpha1
+    kind: ClusterServiceVersion
+    metadata:
+        annotations:
+            features.operators.openshift.io/token-auth-aws: "true"
+            ...
+    ```
 - Add a script in the operator description, per supported cloud provider, to help set up the correct role
 - (Optional) Add the projected ServiceAccount volume to the Deployment embedded in the CSV;
-- handle the Secret themselves (like today’s CCO Mint mode, read and use the credentials in the Secret whose name they 
-  know from the CredentialsRequest and is in the same Namespace)
+- When OLM starts the operator with `ROLEARN` env. variable, handle the Secret themselves (like today’s CCO Mint mode,
+  read and use the credentials in the Secret whose name they know from the CredentialsRequest and is in the same
+  Namespace.)
```

Comment:
aren't we missing the step where the operator is expected to create the CredReq object, also, when the cluster is in STS mode?

Reply:
I assume that when `ROLEARN` is set, then OLM / console has checked CCO is in STS mode already.

I rephrased the operator behavior, now it should be clearer.